scripts/game_structure/load_cat.py

```diff
@@ -3,20 +3,15 @@
 from .game_essentials import game
 from ..datadir import get_save_dir
 
-try:
-    import ujson
-except ImportError:
-    import json as ujson
+import ujson
 
 from re import sub
 from scripts.cat.cats import Cat
 from scripts.cat.pelts import choose_pelt, vit, point_markings
 from scripts.utility import update_sprite, is_iterable
 from random import choice
-try:
-    from ujson import JSONDecodeError
-except ImportError:
-    from json import JSONDecodeError
+
+from json import JSONDecodeError
```

Comment:
Is there no difference between JSONDcodeError from ujson and json? Even so, I suggest to use 
```suggestion
from ujson import JSONDecodeError
```
to be in line with everything

Reply:
ujson does not have a JSONDesoceError, so the code was pulling it from json anyway

[Caused pylint to fail](https://github.com/ImLvna/clangen/actions/runs/4585415188/jobs/8097549533)